### PR TITLE
Add beneficiary and textlockup

### PR DESCRIPTION
**Pull request by drewdrewthis**

### CODE REVIEW

File: src/components/text-lockups/text-lockups.module.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import 'src/assets/stylesheets/base_styles/typography';
+@import 'src/assets/stylesheets/base_styles/colors';
+@import 'src/assets/stylesheets/utilities/functions';
+@import 'src/assets/stylesheets/utilities/mixins';
+
+.type1 {
+  $subheader-color: $neutral-4;
+
+  .header {
```
Comment (jmcolella) [inline src/components/text-lockups/text-lockups.module.scss]: what's the benefit of nesting here? I don't think we need to do this in general, just maybe for pseudo selectors

File: src/modules/cards/cards.module.scss [hunk 2]
```diff
@@ -51,18 +52,98 @@ $hover-color:        $neutral-2;
...
+  position: relative;
+  min-width: ru(11.33);
+  background-color: $beneficiary-card-background-color;
+  margin-bottom: ru(1);
+
+  .mod-percentage-card {
+    display: none;
+  }
+
+  &.collapsed {
```
Comment (jmcolella) [inline src/modules/cards/cards.module.scss]: this should not be nested but instead composed from beneficiary card. `composes` will do the work of `&` behind the scenes.

This opens up the discussion of making individual Sass files for each module/component

File: src/modules/cards/cards.module.scss [hunk 3]
```diff
@@ -1,24 +1,25 @@
-@import 'src/assets/stylesheets/base_styles/typography';
-@import 'src/assets/stylesheets/base_styles/colors';
-@import 'src/assets/stylesheets/utilities/functions';
-@import 'src/assets/stylesheets/utilities/mixins';
+@import 'src/assets/stylesheets/global';
```
Comment (jmcolella) [inline src/modules/cards/cards.module.scss]: smart 👍

Comment (drewdrewthis) [inline src/modules/cards/cards.module.scss]: @jmcolella Kind of.. but do we want to do it explicitly, or do we want it to simply be part of our post-css pipeline?

File: src/modules/cards/cards.module.scss (hunk 2 above)
Comment (drewdrewthis) [inline src/modules/cards/cards.module.scss]: @jmcolella Agreed -- these styles were actually pulled in from our existing repo

File: src/modules/cards/cards.module.scss (hunk 3 above)
Comment (jmcolella) [inline src/modules/cards/cards.module.scss]: @drewdrewthis yeah I had that thought in the back of my mind as well...I'm going to be investigating more into post css today, so we should have a better idea

**Pull request merged**